xwiki-platform-core/xwiki-platform-security/xwiki-platform-security-requiredrights/xwiki-platform-security-requiredrights-default/src/main/java/org/xwiki/platform/security/requiredrights/internal/analyzer/DefaultMacroBlockRequiredRightAnalyzer.java

```diff
@@ -71,6 +77,9 @@ public class DefaultMacroBlockRequiredRightAnalyzer extends AbstractMacroBlockRe
     @Inject
     private Provider<DefaultMacroRequiredRightReporter> macroRequiredRightReporterProvider;
 
+    @Inject
+    private Converter<MacroContentSourceReference> macroContentSourceReferenceConverter;
```

Comment:
Hmm, this field does not seem to be used (which would make sense), but maybe I missed it.

Reply:
Good catch, this was from an earlier version of the implementation where I had the analysis directly in that component. I've removed it.